build/makelib/golang.mk

```diff
@@ -177,6 +183,12 @@ $(GO_TOOLS_DIR)/glide:
 	@mv $(GO_TOOLS_DIR)/$(GOHOSTOS)-$(GOHOSTARCH)/glide $(GO_TOOLS_DIR)/glide
 	@rm -r $(GO_TOOLS_DIR)/$(GOHOSTOS)-$(GOHOSTARCH)
 
+.PHONY: $(GO_TOOLS_DIR)/golint
```

Comment:
not sure I follow why this is a PHONY.  You want the behavior to be that you dont install golint if the file exists just like glide:

https://github.com/rook/rook/blob/master/build/makelib/golang.mk#L173